src/test/ui/proc-macro/auxiliary/nonterminal-recollect-attr.rs

```diff
@@ -10,6 +10,7 @@ use proc_macro::{TokenStream, quote};
 #[proc_macro_attribute]
 pub fn first_attr(_: TokenStream, input: TokenStream) -> TokenStream {
     let recollected: TokenStream = input.into_iter().collect();
+    println!("First recollected: {:?}", recollected);
```

Comment:
```suggestion
    println!("First recollected: {:#?}", recollected);
```
It's hard to read the output when everything is formatted on a single line.